negotiation/src/test/java/it/eng/negotiation/service/ContractNegotiationProviderServiceTest.java

```diff
@@ -152,22 +153,49 @@ public void getNegotiationByProviderPid_notFound() {
     }
     
     @Test
-    public void finalizeNegotiation_success() {
-    	ContractNegotiation cn = ContractNegotiation.Builder.newInstance()
-                .consumerPid(ModelUtil.CONSUMER_PID)
-                .providerPid(ModelUtil.PROVIDER_PID)
-                .state(ContractNegotiationState.ACCEPTED)
-                .build();
-        when(repository.findByProviderPidAndConsumerPid(anyString(), anyString())).thenReturn(Optional.of(cn));
-		when(okHttpRestClient.sendRequestProtocol(any(String.class), any(JsonNode.class), any(String.class))).thenReturn(apiResponse);
-		when(apiResponse.isSuccess()).thenReturn(true);
-
-    	service.finalizeNegotiation(ModelUtil.CONTRACT_AGREEMENT_VERIFICATION_MESSAGE);
+    public void getNegotiationById() {
```

Comment:
Can you add Display annotation with test description, so we are consistent, at least inside this test?

Reply:
No problemo